### PR TITLE
fix(DB/Zone):improved Western Plaguelands

### PR DESCRIPTION
**Pull request by Knindzagxg**
Western Plaguelands are now fully updated, since some creatures in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele to zone and run / test around.

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Si1ker** [comment]
**TEST: 3 failed** 🐱 @FrancescoBorzi @Knindzagxg 

Spectral Attendant - On Aggro - Cast \'11443\''),
image

Spectral Attendant - In Combat - Cast \'13860\''),
image

Spectral Attendant - Between 30-60% Health - Cast \'11981\' (No Repeat)');
image

============================
'Fetid Zombie - In Combat - Cast \'8714\'');
image


============================
Haunting Vision - On Aggro - Cast \'11443\''),
image

Haunting Vision - In Combat - Cast \'13860\''),
image

'Haunting Vision - Between 30-60% Health - Cast \'11981\' (No Repeat)');
image


============================
Decaying Horror - On Aggro - Cast \'32065\'');
image


============================
'Plague Lurker - Between 20-80% Health - Cast \'3436\' (No Repeat)');
image


Huntsman Radley - On Aggro - Summon Creature \'Bloodshot\''),
image


'Huntsman Radley - In Combat - Cast \'11976\''),
image


'Huntsman Radley - Between 20-80% Health - Cast \'14443\' (No Repeat)');
### [FAILED]
 I can't get this one to work. reading the spell description it says:

image

I have tried to run away in order to allow the distance but it just does not happen  : /


============================
'Bloodshot - Between 20-80% Health - Cast \'13692\' (No Repeat)'),
image

'Bloodshot - On Evade - Despawn In 5000 ms');
image

( It's fading away .... )


============================
Scarlet Cavalier - In Combat - Cast \'16856\''),
image



'Scarlet Cavalier - Between 40-80% Health - Cast \'15284\' (No Repeat)'),
image


Scarlet Cavalier - Between 10-30% Health - Cast \'6253\' (No Repeat)');
image


============================
'Scarlet Priest - On Respawn - Cast \'13864\''),
image
### [FAILED]
When the NPC spawns, It will cast 13864 at the first only. If you leave combat with the NPC, the aura will be gone, if you engaged the NPC once again, it will not use it, this is not correct. Take for example the NPC with ID 1889, this NPC uses a shield (12544) that will always use, at reset it recast the spell so the npc is always looking to have it up/active at all time (meaning in combat and out of combat), same way it should be working here. OR use it at agro instead of respawn, your choice.



 'Scarlet Priest - In Combat - Cast \'15498\''),
image


Scarlet Priest - Between 60-90% Health - Cast \'8362\' (No Repeat)'),
image


'Scarlet Priest - Between 5-25% Health - Cast \'12039\' (No Repeat)');
image


============================
'Cavalier Durgen - On Aggro - Cast \'13008\''),
image


 'Cavalier Durgen - In Combat - Cast \'13953\''),
image


 'Cavalier Durgen - Between 10-30% Health - Cast \'13005\' (No Repeat)');
image


============================
'Scarlet Lumberjack - In Combat - Cast \'11977\''),
image


'Scarlet Lumberjack - Between 25-40% Health - Cast \'15496\' (No Repeat)');
image


============================
 'Skeletal Terror - On Aggro - Cast \'17650\''),
image


 'Skeletal Terror - Between 5-30% Health - Cast \'12542\' (No Repeat)');
image

============================
 'Scarlet Medic - On Respawn - Cast \'17175\''),
image
### [FAILED]
When the NPC spawns, It will cast 17175at the first only. If you leave combat with the NPC, the aura will be gone, if you engaged the NPC once again, it will not use it, this is not correct. Take for example the NPC with ID 1889, this NPC uses a shield (12544) that will always use, at reset it recast the spell so the npc is always looking to have it up/active at all time (meaning in combat and out of combat), same way it should be working here. OR use it at agro instead of respawn, your choice.


 'Scarlet Medic - Between 5-95% Health - Cast \'11640\' (No Repeat)'),
image


 'Scarlet Medic - Between 5-20% Health - Cast \'17137\' (No Repeat)');
image


============================
'Jabbering Ghoul - On Aggro - Cast \'15716\''),
image


'Jabbering Ghoul - In Combat - Cast \'12097\'');
image


============================
 'Venom Mist Lurker - In Combat - Cast \'3583\' (No Repeat)');
image

**Comment by Knindzagxg** [comment]
Jesus those tests xD

Updated Priest and Medic as suggested xP Weird it's droping btw, never saw it before.
Updated Auras as suggested as well xP

Gonna look  a bit more into Huntsman, cause he did cast few times on me.

**Comment by Knindzagxg** [comment]
Sorted Huntsman.

For some reason, Hostile_random not working at all.
Tried with multiple chars but no luck at all.
At least its good to know, so we can avoid until its fixed.

Recheck if u can again @Si1ker xD

File: data/sql/updates/pending_db_world/rev_1601314686326215700.sql [hunk 1]
```diff
@@ -40,7 +40,7 @@ DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 11613);
 INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
 (11613, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 12, 11614, 1, 30000, 1, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - On Aggro - Summon Creature \'Bloodshot\''),
 (11613, 0, 1, 0, 0, 0, 100, 0, 2700, 3900, 9700, 12400, 11, 11976, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - In Combat - Cast \'11976\''),
-(11613, 0, 2, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 14443, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - Between 20-80% Health - Cast \'14443\' (No Repeat)');
+(11613, 0, 2, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 14443, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - Between 20-80% Health - Cast \'14443\' (No Repeat)');
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1601314686326215700.sql:43]: @Knindzagxg This one is still not working...  :,c

Just melees and her 11976..
image

File: data/sql/updates/pending_db_world/rev_1601314686326215700.sql [hunk 2]
```diff
@@ -40,7 +40,7 @@ DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 11613);
 INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
 (11613, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 12, 11614, 1, 30000, 1, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - On Aggro - Summon Creature \'Bloodshot\''),
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1601314686326215700.sql:41]: @Knindzagxg  I noticed something interesting this time about this...
She already has one out at normal spawn... so when you get in combat with her, she will execute this and spawn the second one, is still ok that she has 2 of this minion at combat then (the original + the summon)?

The original and permanent one is:
NPC ID: 11614
GUID: 52640
Coords: X= 2328.97  Y= -2070.82  Z= 168.62
image

**Comment by Si1ker** [comment]
Cavalier Durgen 
Scarlet Priest
Scarlet Medic

These are just fine now 🐱

File: data/sql/updates/pending_db_world/rev_1601314686326215700.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1601314686326215700.sql:43]: It's working, just make sure to make small macro and keep doing damage while away more then 5 yards xP

File: data/sql/updates/pending_db_world/rev_1601314686326215700.sql (hunk 2 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1601314686326215700.sql]: Looks like Blizz decided to not use spell for spawn and have it up always.
Sorted it as well.

File: data/sql/updates/pending_db_world/rev_1601314686326215700.sql (hunk 1 above)
Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1601314686326215700.sql]: @Knindzagxg Believe me, is not working..  :'c

image

**Comment by Si1ker** [comment]
Minion summon resolved, now just the multi-shot thing is left. 🐱

**Comment by Knindzagxg** [comment]
It's sorted now xP
Just unlucky procs with yard and hp % xP

**Review by Si1ker — APPROVED** [review]
Let's gooo ! 🐱

**Pull request merged**